tests/countries/test_india.py

```diff
@@ -34,124 +34,428 @@ def test_subdiv_deprecation(self):
 
     def test_2018(self):
         self.assertHoliday(
```

Comment:
These are not all nation-wide holidays in 2018. Also, to check list completeness, it's better to use
```python
self.assertHolidayDates(
    India(years=2018),
    ....
)
```